tests/integration_tests/test_botorch.py

```diff
@@ -97,6 +102,13 @@ def candidates_func(
     ],
 )
 def test_botorch_specify_candidates_func(candidates_func: Any, n_objectives: int) -> None:
+    if candidates_func == integration.botorch.logei_candidates_func and sys.version_info < (
+        3,
+        8,
+        0,
+    ):
```

Comment:
Note: `botorch >=0.8.1` is not available in python 3.7, so I had to skip some tests.

Reply:
Skipping the test based on the Python version seems to be a bit indirect. How about checking the `botorch` version instead?

Reply:
```suggestion
    if candidates_func == integration.botorch.logei_candidates_func and version.parse(botorch.version.version) < version.parse("0.8.1"):
```